README.md

```diff
@@ -665,16 +835,22 @@ import mkdirp from 'mkdirp'
 import fs from 'fs'
 import * as buildCommand from './cli/buildCommand'
 import * as testCommand from './cli/testCommand'
+import * as lintCommand from './cli/lintCommand'
 
 it('works', async () => {
   const example = fs.readFileSync('example.md', 'utf8')
+  const eslintrc = fs.readFileSync('.eslintrc', 'utf8')
   await runInTemporaryDir(async () => {
-    fs.writeFileSync('README.md', example)
+    fs.writeFileSync('README.md', example.replace('a + b', 'a+b'))
+    fs.writeFileSync('.eslintrc', eslintrc)
```

Comment:
There are two possible behaviors:

### 1\. Allow user to supply their own linting rules.

In this case, our project should not declare `eslint` as its dependency, to also allow user to select the ESLint version they want to use.

I believe this is what you are trying to implement.

However, if ESLint is not installed by end-user, this should not cause the CLI to crash (Cannot find module 'eslint'). Instead, when the user tries to use ESLint feature, we should display a friendly message asking the user to install ESLint in their project instead.

Also, a documentation would be nice.

> ### linting
>
> `essay` comes with ESLint integration, which reports ESLint warnings in your
README file and can fix ESLint warnings right inside the README file.
>
> First, you need to install `eslint` and the config you want, for example:
>
> ```
> npm install --save-dev \
>   eslint \
>   eslint-config-standard \
>   eslint-plugin-promise \
>   eslint-plugin-standard
> ```
>
> Create your ESLint config:
>
> ```yaml
> extends: standard
> ```
>
> Then use this command to lint your README file:
>
> ```
> npm run lint
> ```
>
> To fix the lint warnings, run:
>
> ```
> npm run lint-fix
> ```

### 2\. Hardcoded `standard` ESLint preset.

In this case, the user should not have to provide an `.eslintrc`.